docs/api/python.rst

```diff
@@ -51,26 +51,31 @@ and rich debugging and accessibility interfaces built-in.
 
 Data Abstractions
 ------------------------------------
-The Folder, Blob, and File APIs provide a simple interface for storing, recalling, and moving data between
-the user's laptop, remote compute, cloud storage, and specialized storage (e.g. data warehouses).
-They provide least-common-denominator APIs across providers, allowing users to easily specify the actions
-they want to take on the data without needed to dig into provider-specific APIs. We'd like to extend this
-to other data concepts in the future, like kv-stores, time-series, vector and graph databases, etc.
+The Folder APIs provide a simple interface for storing, recalling, and moving data between
+the user's laptop, remote compute, and cloud storage (currently we support `S3` and `GCS`). They provide
+least-common-denominator APIs across providers, allowing users to easily specify the actions
+they want to take on the data without needed to dig into provider-specific APIs.
 
 .. toctree::
    :maxdepth: 1
 
    python/folder
 
-.. toctree::
-   :maxdepth: 1
 
-   python/blob
+For example, you can easily load data on a cluster directly from s3, without needing to bounce the data off
+your laptop.
 
-.. toctree::
-   :maxdepth: 1
+.. code-block:: python
+
+    import runhouse as rh
+
+    # Load a folder which points to a particular directory on S3
+    s3_folder = rh.folder(path"/main-bucket/my-folder", system="s3")
```

Comment:
@carolineechen what do you think of this example? we already have tutorials for functions clusters and modules, figured this would be the most useful way of using a folder for now (though we may end up totally deprecating it)

Reply:
I think the example's fine, but we haven't included examples here / have something similar in the api tutorials so I think we can leave it out.